Gemfile

```
@@ -2,3 +2,4 @@ source "http://rubygems.org"
 
 gemspec
 
+gem 'theforeman-rubocop', '~> 0.1'
```

semver says all .y versions in the 0.y series can be breaking so they suggest to pin to:
```suggestion
gem 'theforeman-rubocop', '~> 0.1.0'
```